source/_docs/terminus/configuration.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Terminus Manual
+subtitle: Configuration File
+terminusconfiguration: true
+terminuspage: true
+terminustoc: true
+type: terminuspage
+layout: terminuspage
+nexturl: terminus/examples/
+previousurl: terminus/plugins/create/
+permalink: docs/terminus/:basename/
+image: terminus-thumbLarge
+---
+
+If you find yourself passing the same options to Terminus repeatedly, consider creating a configuration file to provide default values for common options. The Terminus configuration file lives at `$HOME/.terminus/config.yml`.
+
+## What Can I Configure?
+
+Any command variable normally passed in the form of `--option=VALUE`. Values stored will be available regardless of which alias you use to run the command. Default values stored this way will be overridden by those supplied on the command line.
+
+You can see all the available options for a command using the `terminus help COMMAND`:
+
+![Terminus Help Command Example](/source/docs/assets/images/terminus-help-example.png)
+
+## Examples
+
+```yml
+command:
+  auth:
+    login:
+      options:
+        email: anita@example.org
+  site:
+    pancakes:
+      options:
+        app: sequelpro
+```
+
+The example above does two things:
```

Comment:
Config option hierarchy is explained here: https://github.com/consolidation/config#providing-command-options

Feel free to copy in any of that text, or just link there

Reply:
Are we using that repositories code/tools in Terminus to make this work?

Reply:
Yes. Terminus is built on top of Robo. Robo uses consolidation/config for its config system. Terminus' config inherits from Robo's config.